devel/bazel/Portfile

```diff
@@ -1,17 +1,17 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem                                            1.0
-PortGroup             github                          1.0
-PortGroup             java                            1.0
-PortGroup             compiler_blacklist_versions     1.0
-PortGroup             legacysupport                   1.1
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           java 1.0
+PortGroup           compiler_blacklist_versions 1.0
+PortGroup           legacysupport 1.1
```

Comment:
Lots of unrelated whitespace changes in this PR, making the functional changes more difficult to identify and review.

Reply:
The `bazel`-related ports require a complete refactoring, either with the approach used in this PR, or rewritten from scratch using `bazelisk`.

With justification, @cjones051073 threw in the towel on contributing to anything `bazel`-related because of the unreasonable resources necessary to address its breaking changes.

I am in this PR trying to streamline the port the achieve an up-to-date`bazel` version  that builds `tensorflow`, which, by the way, still doesn't build on macOS: https://github.com/tensorflow/tensorflow/issues/60191

I'd recommend viewing this effort as a full rewrite of the port using the existing approach.